### DISCUSSION TITLE
What should I do to deform/move mesh

### DISCUSSION THREAD

**Discussion by Eddudos** [facebookresearch/pytorch3d#1764] [Q&A] [answered]
Hi! I'm trying to figure out if there is any functionality in a pytorch3d that could help me link my mesh (clothes) with another mesh (human). So if I deform my human mesh and the clothes naturally deform too. I need kind of similar linking that is implemented in blender through bones/rigging. 

Also, could you recommend me some specialized forums or telegram channel so I could seek for help there.

**Comment by bottler** [comment, marked as answer]
This functionality does not exist in PyTorch3D.

**Reply by Eddudos** [reply]
Thank you for your response!

So my options are to either implement required functionality myself or train model to fit/deform mesh related to human mesh right?
Is the second option even expedient? Do you think it could it run in close to real-time scenario?

**Reply by bottler** [reply]
There may be other options, perhaps using other libraries. I can't help.
